README.md

```diff
@@ -7,12 +7,12 @@ This proposal does not address the source of the contacts. Many operating system
 
 ## Use cases
   * Social networks could use contact information to bootstrap a user's social graph. This is particularly important for emerging markets, where users might prefer a PWA over a native app due to storage constraints.
-  * An e-mail application could allow the user to select the recipients for a message without needing their own address book.
+  * An e-mail application could allow the user to select recipients for a message without needing its own address book.
 
 ## Why do we need a new built-in API?
   * Excluding the operating system, there is no canonical place where users maintain a list of their contacts. Developers would have to integrate with a variety of services to provide a comprehensive experience.
-  * User agents can offer an experience to users that's consistent with other applications on the user's device. The information that will is going to be shared with the website can be clearly highlighted.
-  * Contact information is sensitive. Individual contacts can have [any number of properties](https://en.wikipedia.org/wiki/VCard#Properties) associated with them. Websites seeking the e-mail address of a single user should not have access to the same information of all other contacts.
+  * User agents can offer an experience to users that's consistent with other applications on the user's device. The information that will be shared with the website can be clearly highlighted.
+  * Contact information is sensitive. Individual contacts can have [any number of properties](https://en.wikipedia.org/wiki/VCard#Properties) associated with them. Websites seeking the e-mail address of a single user should not have access to all other contacts.
 
```

Comment:
**Omit needless words** - If a word can be removed while the sentence retains its meaning, then the word didn't need to be there. If deletion changes the meaning, then said meaning was probably not properly conveyed and the sentence needs to be rewritten.